ccm_web/client/src/components/MultipleUserEnrollmentWorkflow.tsx

```diff
@@ -238,8 +238,9 @@ export default function MultipleUserEnrollmentWorkflow (props: MultipleUserEnrol
 
     const description = (
       'This tool will enroll non-UM users in the selected section. ' +
-      'If they do not have an account in Canvas, they will be sent an email ' +
-      'invitation to choose a login method and added to Canvas before they are enrolled in the section.'
+      'If they do not have an account in Canvas,  an account will be created for them and  ' +
+      'they will be sent an email invitation with instructions how to login before they are enrolled in the section.' +
```

Comment:
```suggestion
      'they will be sent an email invitation with instructions how to login before they are enrolled in the section. ' +
```
Nitpick: add a space, keeping the formatting the same as other sentences in the description